.travis.yml

```diff
@@ -1,4 +1,13 @@
 language: node_js
+env:
+    - CXX=g++-4.8
+addons:
```

Comment:
It is a necessary change, described in [Travis documentation](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements)